src/api/app/models/cloud/upload_job.rb

```diff
@@ -0,0 +1,37 @@
+module Cloud
+  class UploadJob
+    include ActiveModel::Validations
+    include ActiveModel::Model
+    extend Forwardable
+
+    attr_accessor :user, :params
+    attr_reader :user_upload_job, :backend_upload_job
+    validate :validate_jobs
+    def_delegator :backend_upload_job, :id
+
+    def create
```

Comment:
Couldn't we just define an initializer instead? I briefly checked the code and didn't find any case where we just call UploadJob.new

Reply:
@evanrolfe wanted to avoid to override the default initialize method. And we will have in the future something like this ```UploadJob.new(user, params).all``` so using new would be confusing. Also create reflects it better what we're actually doing - creating a job in the backend.